### PR TITLE
[docs](routine-load)Update routine-load-manual.md

### PR DESCRIPTION
**Pull request by sodamnsure**
add stop instructions

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luzhijing** [comment]
You can update the EN docs together.

**Comment by luzhijing** [comment]
LGTM

File: docs/zh-CN/administrator-guide/load-data/routine-load-manual.md [hunk 1]
```diff
@@ -246,7 +246,7 @@ FE 中的 JobScheduler 根据汇报结果，继续生成后续新的 Task，或
 
 ### 作业控制
 
-用户可以通过 `STOP/PAUSE/RESUME` 三个命令来控制作业的停止，暂停和重启。可以通过 `HELP STOP ROUTINE LOAD;`, `HELP PAUSE ROUTINE LOAD;` 以及 `HELP RESUME ROUTINE LOAD;` 三个命令查看帮助和示例。
+用户可以通过 `STOP/PAUSE/RESUME` 三个命令来控制作业的停止(FE会自动定期清理停止的ROUTINE LOAD)，暂停和重启。可以通过 `HELP STOP ROUTINE LOAD;`, `HELP PAUSE ROUTINE LOAD;` 以及 `HELP RESUME ROUTINE LOAD;` 三个命令查看帮助和示例。
 
 ## 其他说明
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/load-data/routine-load-manual.md:252]: 关于 STOP状态的 routine laod job 会被定期清理的解释，建议在其他说明中单独新增一项单独解释。

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**